src/test_utils.rs

```diff
@@ -34,6 +35,13 @@ use crate::mimeparser::MimeMessage;
 #[allow(non_upper_case_globals)]
 pub const AVATAR_900x900_BYTES: &[u8] = include_bytes!("../test-data/image/avatar900x900.png");
 
+/// `tcm.sec()` adds info events that mark a section in the log, e.g.:
+///
+/// ========== Example section ==========
+///
+/// This is the id for these events.
+const SEC_EVENT_ID: u32 = 0;
```

Comment:
There is no need for this constant, it is only used once. And usage of 0 context ID for events is not limited to sections, account manager uses it too to log its errors.